docs/components/carousel.html

```diff
@@ -0,0 +1,127 @@
+                        <a class="carousel-control-prev" href="#carouselWithControls" role="button" data-slide="prev">
+                            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+                            <span class="sr-only">Previous</span>
+                        </a>
+                        <a class="carousel-control-next" href="#carouselWithControls" role="button" data-slide="next">
+                            <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                            <span class="sr-only">Next</span>
+                        </a>
+                    </div>
+                </div>
+            </div>
+
+            <h1>CAROUSEL WITH GROUPED CONTROLS</h1>
+            <div class="row mb-3">
+                <div class="col">
+                    <div id="carouselWithGroupedControls" class="carousel slide" data-ride="carousel">
+                        <ol class="carousel-indicators">
+                            <li data-target="#carouselWithGroupedControls" data-slide-to="0" class="active rounded-circle border-0"></li>
+                            <li data-target="#carouselWithGroupedControls" data-slide-to="1" class="rounded-circle border-0"></li>
+                            <li data-target="#carouselWithGroupedControls" data-slide-to="2" class="rounded-circle border-0"></li>
+                        </ol>
+                        <div class="carousel-inner">
+                            <div class="carousel-item active">
+                                <img class="d-block w-100" src="https://via.placeholder.com/300x100" alt="First slide">
+                            </div>
+                            <div class="carousel-item">
+                                <img class="d-block w-100" src="https://via.placeholder.com/300x100" alt="Second slide">
+                            </div>
+                            <div class="carousel-item">
+                                <img class="d-block w-100" src="https://via.placeholder.com/300x100" alt="Third slide">
+                          </div>
+                        </div>
+                        <a class="carousel-control-prev carousel-control-prev--grouped-right-desktop-only" href="#carouselWithGroupedControls" role="button" data-slide="prev">
+                            <span class="carousel-control-prev-icon carousel-control-prev-icon--background-white" aria-hidden="true"></span>
+                            <span class="sr-only">Previous</span>
+                        </a>
+                        <a class="carousel-control-next carousel-control-next--grouped-right-desktop-only" href="#carouselWithGroupedControls" role="button" data-slide="next">
+                            <span class="carousel-control-next-icon carousel-control-next-icon--background-white" aria-hidden="true"></span>
+                            <span class="sr-only">Next</span>
+                        </a>
```

Comment:
classes here can be optimised.
the only difference with the default control is the background, so we add a class for the background on main control element (not the child, please see my comments on the scss) and class for the right position (please always refer to devices as breakpoints, so it must be lg)
see this code:
```
<a class="carousel-control-prev carousel-control--background  carousel-control--right-lg" href="#carouselWithGroupedControls" role="button" data-slide="prev">
                            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
                            <span class="sr-only">Previous</span>
                        </a>
                        <a class="carousel-control-next carousel-control--background  carousel-control--right-lg" href="#carouselWithGroupedControls" role="button" data-slide="next">
                            <span class="carousel-control-next-icon" aria-hidden="true"></span>
                            <span class="sr-only">Next</span>
                        </a>
```